go-controller/pkg/cni/cnishim.go

```diff
@@ -118,17 +158,21 @@ func (p *Plugin) CmdAdd(args *skel.CmdArgs) error {
 	if err != nil {
 		return fmt.Errorf("invalid stdin args")
 	}
+	setupLogging(conf)
```

Comment:
yes, because we exec the binary for every call to CNI_ADD

Reply:
I think you could simplify this patch a bunch by moving most of it to `cmd/ovn-k8s-cni-overlay/ovn-k8s-cni-overlay.go`. Set up logging from `main` there, use `skel.PluginMainWithError` rather than `skel.PluginMain` so you get the error return value of the `Cmd` functions, and `klog.Error` if you get back an error.

Reply:
@danwinship I started at main and then moved it where it is now. When it was  inmain I had to do this

1. I need to read `os.Stdin` to find out the logFile and logLevel
2. After having read `os.Stidn` I need to write the data back to `os.Stdin` since the libCNI reads off of it to read NetConf

BTW, it is not only error messages that I would like to log but also various debug messages from ovn-kubernetes/go-controller/pkg files